src/collection/class.js

```diff
@@ -5,13 +5,13 @@ let elesfn = ({
     classes = ( classes || '' ).match( /\S+/g ) || [];
     let self = this;
     let changed = [];
-    let classesMap = {};
+    let classesMap = new Set();
```

Comment:
You need to `require()` the `set.js` file under `/src` so the polyfill is used in case the code is running in an old environment where `Set` is not supported.